python/tritonserver/_api/_response.py

```diff
@@ -182,9 +181,6 @@ def _response_callback(self, response, flags, unused):
                 asyncio.run_coroutine_threadsafe(
                     self._user_queue.put(response), self._loop
                 )
-            if flags == TRITONSERVER_ResponseCompleteFlag.FINAL:
-                del self._request
-                self._request = None
```

Comment:
From bindings / lifecycle perspectives, are there any cons introduced by **not** deleting the request here or somewhere else? Will an object get kept around unexpectedly?

Reply:
I added `while True` but didn't notice any memory increase.

Reply:
Memory usage appears to be stable. Might need stress tests to make sure it is not leaking any memory.